src/web/crate_details.rs

```diff
@@ -237,6 +254,31 @@ impl CrateDetails {
 
         Some(crate_details)
     }
+
+    /// Returns all versions of this crate.
+    pub fn versions(&self) -> Vec<String> {
+        self.releases.iter()
+            .map(|release| release.version.clone())
+            .collect()
+    }
+}
+
+
+fn map_to_release(conn: &Connection, crate_id: i32, version: String) -> Release {
+    let rows = conn.query(
+        "SELECT build_status
+         FROM releases
+         WHERE releases.crate_id = $1 and releases.version = $2;",
+        &[&crate_id, &version],
+    ).unwrap();
+
+    let build_status = if rows.len() >= 1 {
```

Comment:
```suggestion
    let build_status = if !rows.is_empty() {
```

Reply:
Good eye, fixed it!

Reply:
Blame clippy, not me :)

Reply:
Hmm, weird. Clippy isn't saying anything on my machine... 😕